### ISSUE TITLE
extra="true" in profile.conf ignored

### ISSUE THREAD

**Issue by oberon2007** [manjaro/manjaro-tools#289]
@fhdk @Ste74 seems your latest modifications have created a regression:
I just built my three community ISOs, deepin, cinnamon and i3 and only realized afterwards that they have been built as 'minimal' ISOs even though in the profile.conf they all have specified:
`extra="true"`

**Comment by Ste74** [comment]
@oberon2007 https://github.com/manjaro/manjaro-tools/pull/286
this is right but only if you use the -git version .. if you use the normale release tagged by @philmmanjaro no. I don't remove in the iso-profiles repo the extra tag because i refer at the stable release .. when m-t update the release we remove the flag .. ;)

**Comment by Ste74** [comment]
Ohhho.. @Chrysostomus  why use m-t git version ? it create the minimal version .. maybe we have wait the next m-t release to use the -f flag also because the isos are called in different mode .. no ?

**Comment by oberon2007** [comment]
I always use the latest git-snapshot of stable branch as it reflects our current state of development.
My understanding was that you have added an override flag -f to build full ISOs without the need to edit a config file. I was not aware that the setting in the config file is now ignored completely.
Is this really what we want?
Acutally I always build full profiles - so now I always have to use the -f flag and I don't have a way to set this permanently?
Why can we not keep both?

**Comment by Ste74** [comment]
I'm open at all .. we can use it in the same way for override the kernel :)

**Comment by oberon2007** [comment]
My understanding during our discussion was that what the -f flag does is simply set extra="true" - sounded quite simple but apparently this went a different way later ... ;)

**Comment by oberon2007** [comment]
In any case I am now building all the ISOs again - with the -f flag added... :stuck_out_tongue_closed_eyes:

**Comment by Ste74** [comment]
We have only have know what we want .. if for anyone of us is usefull leave the extra=true in the profile.conf why not ..

**Comment by philmmanjaro** [comment]
I'm still using the packaged tools as the new options were added after the final release cycles. The intention was to focus on minimal builds in test cycles and only use the -f flag for RCs. Based on the code `extra="true"` is now ignored. This _regression_ was added with https://github.com/manjaro/manjaro-tools/commit/d266a28a0bc4744a682f3ae922c8b9adc9ff146f. See also [line 436 - 440](https://github.com/manjaro/manjaro-tools/commit/d266a28a0bc4744a682f3ae922c8b9adc9ff146f#diff-f1f0aaee6fe02e534e572e6ed87e8ba7L436).

**Comment by fhdk** [comment]
I have only made a suggestion - as my working on the openbox basic profile progressed I became annoyed that I had to change the profile.conf instead of specifying on cli if I wanted a full or a minimal iso.

Initially I thought of a way to override the extra=true but @Ste74 told me that initially the scripts were made to build minimal and setting extra on the profile was a workaround.

I then modified the change so a flag created a full iso and without the flag a minimal iso was build.

As @Ste74 said when I created the pr - it should make the `extra=` obsolete as it initially was a workaround. So I did change it so the setting in the profile is obsolete.

So it is not a regression but fully thought through change to the default behavior as I understood @Ste74
